src/GoodsList.tsx

```diff
@@ -8,7 +9,15 @@ type Props = {
 export const GoodsList: React.FC<Props> = ({ goods }) => (
   <ul>
     {goods.map(good => (
-      <li key={good.id} data-cy="good">
+      <li
+        className={classNames(
```

Comment:
I think you don't need to use css for this case. You can use only className and key: value whis goods list object
`style={{ color: good.color }}`